Example-iOS/Podfile

```diff
@@ -0,0 +1,9 @@
+source 'https://github.com/CocoaPods/Specs.git'
+
+target 'VOKBenkode', :exclusive => true do
```

Comment:
It wouldn't be added directly to the test target, true. But shouldn't it be available from the target dependency?